File: ATMMACHINE/src/main/Card.java
package main;

public class Card {

    int pin;
    public int getPin() {
        return pin;
    }

    public void setPin(int pin) {
        this.pin = pin;
    }

    BankAccount account;

    public Card(int pin, BankAccount account) {
        this.pin = pin;
        this.account = account;
    }
}

File: ATM_CPP/src/Card.cpp
#include "Card.hpp"

Card::Card(std::string cardNumber, std::string customername,  int cardExpiry, int pin)
    : cardNumber(cardNumber), customerName(customerName), cardExpiry(cardExpiry), pin(pin) {}

std::string Card::getCardNumber() const {
    return cardNumber;
}

int Card::getPin() const{
    return pin;
}